google/cloud/README.md

```diff
@@ -20,7 +20,19 @@ Documentation for the common utilities is available [online][doxygen-link].
 
 ## Release Notes
 
-### v0.11.x - TBD
+### v0.12.x - TBD
+
+### v0.11.x - 2019-09
+
+* feat: Use macros for compiler id and version (#2937)
+* fix: Fix bazel build on windows. (#2940)
+* chore: disabled google-runtime-int (#2958)
+* chore: Keep release tags in master branch. (#2963)
+* cleanup: Use only find_package to find dependencies. (#2967)
+* chore: Configure tsan and msan builds. (#3011)
```

Comment:
We can probably remove that too, but up to you.

Reply:
Done.